### PR TITLE
Updated Lazy document to handle viewing PDF's without downloading.

### PR DESCRIPTION
**Pull request by lwjameson**
## In this PR

* Added ViewPDFButton which accepts a redirect URL to view a PDF
* Refactored LazyDocument to include an option to allow viewing of a PDF
* Refactored LazyDocument to not use CSS files imported from node_modules (https://nextjs.org/docs/messages/css-npm)
  * Not really sure if this is the best approach to get around this issue but have not found anything else that will successfully build the NextJS app.

### CODE REVIEW

File: packages/semantic-ui/src/components/FacetSlider.js [hunk 1]
```diff
@@ -72,12 +76,12 @@ const FacetSlider = forwardRef(({ useRangeSlider, ...props }: Props, ref: HTMLEl
           columns={2}
         >
           <Grid.Column>
-            { valueView[0] }
+            {valueView[0]}
```
[the remark this answers is not in the document]
Comment (lwjameson) [reply packages/semantic-ui/src/components/FacetSlider.js:79]: Do we have a style guide I can use in VSCode? This I believe is the default ruleset of the VSCode formatter it is using as I do not have prettier activated right now...Though I think we should use it ;-)

File: packages/semantic-ui/src/components/LazyDocument.js [hunk 2]
```diff
@@ -26,6 +31,8 @@ type Props = {
   image?: any,
   pdf?: boolean,
   preview?: ?string,
+  view?: Boolean,
+  view_url?: String,
```
[the remark this answers is not in the document]
Comment (lwjameson) [reply packages/semantic-ui/src/components/LazyDocument.js:35]: Updated.

File: packages/semantic-ui/src/components/ViewPDFButton.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  url: string,
+};
+
+const ViewPDFButton = (props: Props) => {
+  /**
+   * Sets the appropriate class names based on the formatting props.
+   *
+   * @type {string}
+   */
+  const className = useMemo(() => {
```
[the remark this answers is not in the document]
Comment (lwjameson) [reply packages/semantic-ui/src/components/ViewPDFButton.js:24]: This is a copy/paste of the DownloadButton component as a starting point.  I switched from using an anchor to see if I could resolve an earlier issue.  I use this button outside of LazyDocument in the ArchNet client so I think it makes sense to keep it as a functional component, but I could definitely refactor the classname stuff.  Do not know if it is a priority right now.

File: packages/semantic-ui/src/components/AnnotationLayer.css [hunk 4]
```diff
@@ -0,0 +1,329 @@
+/* Copyright 2014 Mozilla Foundation
```
[the remark this answers is not in the document]
Comment (lwjameson) [reply packages/semantic-ui/src/components/AnnotationLayer.css:1]: That did work.  Updating.

**Review by dleadbetter — APPROVED** [review]

**Pull request merged**